section4/exercises/burrito.rb

```diff
@@ -11,9 +11,30 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_topping(extra_topping)
+    @toppings = toppings.push(extra_topping)
```

Comment:
Nice job making some dynamic functions here, using parameters. You've also used the piece of syntax missing from your superhero!